cli/cli/Commands/Project/GroupCommand.cs

```diff
@@ -0,0 +1,55 @@
+using Beamable.Common.Semantics;
+using Spectre.Console;
+using System.CommandLine;
+
+namespace cli.Commands.Project;
+
+public class GroupCommand : CommandGroup
+{
+	public static Argument<List<string>> GroupsArgument = new("groups", "Beamable groups for this service")
+	{
+		Arity = ArgumentArity.ZeroOrMore,
+		
+	};
+	public GroupCommand() : base("group", "List Service Groups")
+	{
+	}
+
+	public override void Configure()
+	{
+	}
+
+	public override Task Handle(CommandGroupArgs args)
+	{
+		var table = new Table();
+		table.AddColumn("Group name");
+		table.AddColumn("Services IDs");
+		foreach (var record in args.BeamoLocalSystem.BeamoManifest.ServiceGroupToBeamoIds)
+		{
+			table.AddRow(record.Key, string.Join(", ", record.Value));
+		}
+		if (args.BeamoLocalSystem.BeamoManifest.ServiceGroupToBeamoIds.Count > 0)
+		{
+			AnsiConsole.Write(table);
+		}
+		else
+		{
+			AnsiConsole.WriteLine("No services groups found");
+		}
+
+		return Task.CompletedTask;
+	}
+}
+
+
+public class UpdateGroupArgs : CommandArgs
```

Comment:
In general, try to avoid using API-facing types (`CommandArgs`) as the same type in the internal system's API there. Interface types have all sorts of backward compatibility concerns that internal systems don't. Its less likely for people to accidentally break things if these are separate types (even if their schemas are mostly the same, semantically they mean very different things --- the API one means "this is how users interact with the CLI" and the internal system means "these are the ways we can modify groups all through our code-base").